samples/monster_generated.h

```diff
@@ -510,6 +511,14 @@ inline bool VerifyEquipment(flatbuffers::Verifier &verifier, const void *obj, Eq
   }
 }
 
+inline bool VerifyEquipmentVector(flatbuffers::Verifier &verifier, const flatbuffers::Vector<flatbuffers::Offset<void>> *values, const flatbuffers::Vector<uint8_t> *types) {
+  assert(values->size() == types->size());
```

Comment:
since the sizes of these arrays are created by whoever builds these arrays, returning false is better than an assert. It be nice to add an assert in the builder code, but I am not sure where.